Sources/Rendering/OpenGL/CellArrayBufferObject/index.js

```diff
@@ -11,29 +11,29 @@ const { vtkDebugMacro, vtkErrorMacro } = macro;
 // Static functions
 // ----------------------------------------------------------------------------
 
-function computeInverseShiftAndScaleMatrix(shiftArray, scaleArray) {
+function computeInverseShiftAndScaleMatrix(arrayShift, arrayScale) {
```

Comment:
now that I understood the whole thing, it is really a "shift" array of size 3 and not really the "shifts" of the (point data) array.  Sorry about that.

Maybe "shifts" and "scales"  would be better anyway :-)